lib/python/Components/Converter/TrackInfo.py

```diff
@@ -95,7 +96,7 @@ def getText(self):
 					kodi = KodiVideoPlayer.instance
 				except:
 					kodi = None
-				if kodi and fileExists("/usr/lib/enigma2/python/Plugins/Extensions/SubsSupport/plugin.pyo"):
+				if kodi and fileExists("/usr/lib/enigma2/python/Plugins/Extensions/SubsSupport/plugin.", getPyExt()):
```

Comment:
Are you sure about "string" , getPyExt().
This looks like a second parameter for fileExists.

Reply:
No I'm not sure as I can't test it because we don't have it in OV.

We don't use hard-coded path and I need your help testing it and give me the correct syntax.

Once I get the what's right I can fix them all :+1:

Reply:
Please let all the py files as they are.